tools/plisttool/plisttool.py

```diff
@@ -1176,6 +1181,14 @@ def _validate_entitlements_against_profile(
                 self.target, aps_environment, profile_aps_environment),
               **report_extras)
 
+    beta_reports_active = entitlements.get('beta-reports-active')
+    profile_key = (profile_entitlements or {}).get('beta-reports-active')
+    if profile_key != beta_reports_active:
```

Comment:
Your comment in the cl description:

> This is a case where having it in the profile at all requires it's in your entitlements.

Seems like it should be recorded here at the checks; otherwise, the fact that this is different than the APS one above in handing it missing can look like a bug.

To that point, I think we should generate a specific message for when it is missing rather than saying _None_ doesn't match _True_, as other developers might not realize it works differently and has to be there.

Reply:
@keith were you still interested in finishing this out?

Reply:
Yea sorry, haven't had a chance to get back to it

Reply:
Updated, let me know if that's not what you were thinking

Reply:
Thanks